sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Configs.java

```diff
@@ -456,6 +460,20 @@ public int getGlobalEndpointManagerMaxInitializationTimeInSeconds() {
         return getJVMConfigAsInt(GLOBAL_ENDPOINT_MANAGER_INITIALIZATION_TIME_IN_SECONDS, DEFAULT_GLOBAL_ENDPOINT_MANAGER_INITIALIZATION_TIME_IN_SECONDS);
     }
 
+    public static String getAadScopeOverride() {
```

Comment:
You don't need to change the implementation of this method, it can be exactly what you had earlier, but just use the property variable when reading property and env variable when reading env.